core/src/test/java/org/ehcache/EhcacheBasicCrudBase.java

```diff
@@ -730,6 +731,13 @@ public void describeTo(final Description description) {
      * Keys for which access results in a thrown {@code Exception}.  This set may be empty.
      */
     private final Set<String> failingKeys;
+    
+    /**
+     * Keys for which success should be reported in {@link BulkCacheLoadingException}
+     */
+    private Set<String> successKeys;
+    
+    private boolean isBulkCacheLoadingExceptionEnabled = false;
```

Comment:
Now using failingKeys for BulkCacheLoadingException. isBulkCacheLoadingExceptionEnabled  flag is required to identify whether failingKeys are for general loading exception(ie RuntimeException) vs BulkCacheLoadingException